### PR TITLE
Added methods to generate Universally Unique Identifiers (UIIDs),

### PR DESCRIPTION
**Pull request by jsfan3**
Added methods to generate Universally Unique Identifiers (UIIDs), that aim to be unique.
Detailed explanation of the working logic is in the Javadoc.

### CODE REVIEW

File: CodenameOne/src/com/codename1/io/Util.java [hunk 1]
```diff
@@ -2217,4 +2217,193 @@ public void run() {
...
+     * example.</p>
+     *
+     * <p>
+     * Code example:</p>
+     * <script src="https://gist.github.com/jsfan3/2fdc5fae2b723cba40e65faab923e552.js"></script>
+     *
+     * @return a pseudo-random Universally Unique Identifier in its canonical
+     * textual representation
+     */
+    public static String getUIID() {
```
Comment (codenameone) [inline CodenameOne/src/com/codename1/io/Util.java:2263]: Should be `UUID` not `UIID`

File: CodenameOne/src/com/codename1/io/Util.java [hunk 2]
```diff
@@ -2217,4 +2217,193 @@ public void run() {
...
+    /**
+     * Creates a custom UUID, from the given two <code>long</code> values.
+     *
+     * @param time the upper 64 bits
+     * @param clockSeqAndNode the lower 64 bits
+     *
+     * @return a Universally Unique Identifier in its canonical textual
+     * representation
+     */
+    public static String getUIID(long time, long clockSeqAndNode) {
```
Comment (codenameone) [inline CodenameOne/src/com/codename1/io/Util.java:2276]: Same

File: CodenameOne/src/com/codename1/io/Util.java (hunk 1 above)
Comment (jsfan3) [reply CodenameOne/src/com/codename1/io/Util.java]: Done :)

File: CodenameOne/src/com/codename1/io/Util.java (hunk 2 above)
Comment (jsfan3) [reply CodenameOne/src/com/codename1/io/Util.java]: Done :)

**Comment by codenameone** [comment]
Thanks!

**Comment by shai-almog** [comment]
@all-contributors please add @jsfan3  for code

**Pull request merged**